### PR TITLE
Add NatSpec for public state variables

### PR DESCRIPTION
**Pull request by fulldecent**
# Blockers

- [ ] #3418 Allow NatSpec comments for variables

### CODE REVIEW

File: docs/natspec-format.rst [hunk 1]
```diff
@@ -26,7 +26,9 @@ Documentation Example
 =====================
 
 Documentation is inserted above each ``class``, ``interface`` and
-``function`` using the doxygen notation format.
+``function`` using the doxygen notation format. Note: a ``public``
```
Comment (bshastry) [inline docs/natspec-format.rst]: Perhaps start the sentence beginning with `Note: ` on a new line?

Comment (fulldecent) [reply docs/natspec-format.rst]: fixed in 99086d07bc4632d8e3f922f4e28db8b6cb08dbfa

**Review by bshastry — DISMISSED** [review]
LGTM.

**Comment by chriseth** [comment]
Do you intend to implement this?

**Comment by Marenz** [comment]
Can you squash the two commits?

**Comment by leonardoalt** [comment]
@fulldecent is the intention to implement that on this PR? Or just update the documentation?

**Comment by chriseth** [comment]
Closing this for now, since it is not really helpful to have it without implementation.

**Comment by leonardoalt** [comment]
I guess this now depends on https://github.com/ethereum/solidity/pull/8532

**Comment by chriseth** [comment]
This is included in #8532 now

**Pull request closed without merging**